### PR TITLE
Use templates

### PR DESCRIPTION
**Pull request by dodecaphonix**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Teamwork</title>
+</head>
+<body>
+    <h1>Hello, {{ who }}!</h1>
+</body>
+</html>
```
Comment (dodecaphonix) [inline templates/index.html]: Oh my bad. I forgot the newline. I'll push another commit.

**Review by BlueSkies503 — APPROVED** [review]
looks good to me

File: routes.py [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+)
 
 app = Flask(__name__)
 
 
 @app.route("/")
-def hello_world():
-    return "<h1>Hello, World!</h1>"
+def index():
+    return render_template('index.html', who="World")
```
Comment (BlueSkies503) [inline routes.py:11]: What does the second argument in this line mean? `render_template('path', who=<"What's this?">)`

File: routes.py [hunk 3]
```diff
@@ -1,11 +1,14 @@
-from flask import Flask
+from flask import (
+    Flask,
+    render_template,
+)
 
 app = Flask(__name__)
 
 
 @app.route("/")
```
Comment (BlueSkies503) [inline routes.py:9]: Why the `@` symbol? Are we telling computer how to delineate folders in the path?

**Review by BlueSkies503 — COMMENTED** [review]
Some questions about the code.

File: routes.py (hunk 3 above)
Comment (dodecaphonix) [reply routes.py:9]: It's a bit advanced, but they are called "Decorators" and they modify the behavior of a method or function. In this case they are letting flask know that this is the method to call when someone navigates to whatever path is in that string.

Here's a primer on decorators, but don't worry about it yet: https://realpython.com/primer-on-python-decorators/

File: routes.py (hunk 2 above)
Comment (dodecaphonix) [reply routes.py:11]: Good question! I was hoping you'd ask that. That basically says that any variables with the name `who` in the template should have the value "World". We could pass anything in there and change who we are saying hello to. It's kind of like a format string, but for the HTML.

**Pull request merged**